payment-service/src/main/java/org/innowise/paymentservice/model/entity/Payment.java

```diff
@@ -0,0 +1,27 @@
+package org.innowise.paymentservice.model.entity;
+
+import lombok.Data;
+import org.innowise.paymentservice.model.PaymentStatus;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+
+@Data
+@Document
```

Comment:
Missing collectionName attribute in @Document annotation
Should be: @Document(collection = "payments") to explicitly match the Liquibase collection name